src/main/java/cam72cam/immersiverailroading/thirdparty/CommonAPI.java

```diff
@@ -50,26 +102,122 @@ public FluidStack getFluid() {
         return null;
     }
 
+    public Object[] getProperty(String PropertyName) {
+        return getProperty(PropMap.get(PropertyName));
+    }
+
+    public Object[] getProperty(stockProperty PropertyName) {
```

Comment:
I like your pattern of
if (isinstance) {
  stuff for that instance
}

Let's make sure that's consistent through the function